### PR TITLE
Feature/ui crud users

### PR DESCRIPTION
**Pull request by rohmad-st**
- Add Manage Users Mockup Page 11
- Fix Typo type field in form

### CODE REVIEW

File: views/dashboard/approve.pug [hunk 1]
```diff
@@ -74,7 +74,7 @@ block content
       label.col-sm-3.control-label(for='dob') DOB*
       .col-sm-7
         input.form-control(
-          type='dob',
+          type='text',
```
Comment (kuriakinzeng) [inline views/dashboard/approve.pug]: Shall we also integrate date selector with this input?

File: views/dashboard/approve.pug [hunk 2]
```diff
@@ -107,7 +107,7 @@ block content
       label.col-sm-3.control-label(for='membershipExpiry') Membership/Expiry*
       .col-sm-7
         input.form-control(
-          type='membershipExpiry',
+          type='text',
```
Comment (kuriakinzeng) [inline views/dashboard/approve.pug]: Date selector?

**Comment by rohmad-st** [comment]
Hello @kuriakinzeng I just pushed my changes,
Related commit to aeb41a97e1d64c74c741c01088105f0da8fc1dd3

**Comment by kuriakinzeng** [comment]
Thanks. Can you also add a date picker now?

**Comment by rohmad-st** [comment]
Date picker in DOB field? Okay I'll do

**Comment by rohmad-st** [comment]
Okey I just added Date Picker @kuriakinzeng

**Comment by kuriakinzeng** [comment]
Thanks. The datepicker should be in DD/MM/YYYY. Right now it is MM/DD/YYYY

**Comment by rohmad-st** [comment]
Okey wait a sec @kuriakinzeng

**Comment by rohmad-st** [comment]
Done @kuriakinzeng

**Pull request merged**